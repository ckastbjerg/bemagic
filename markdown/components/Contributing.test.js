/* eslint-env jest */
const Contributing = require('./Contributing');

test('it works', () => {
    expect(Contributing()).toMatchSnapshot();
});
