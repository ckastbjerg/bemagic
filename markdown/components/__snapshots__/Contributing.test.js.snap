exports[`test it works 1`] = `
"
        
        <a name=Contributing></a>
        #### Contributing
    ;
        
            Please view the **[contribution guidelines](../CONVENTIONS.md)** before
            modifying this component
        .
    "
`;
